tests/src/source/NewExpectSyntax.spec.bs

```diff
@@ -164,6 +164,47 @@ namespace tests
       m.assertRunningTestIsPassed()
     end function
 
+    '+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    @describe("stubCall local and namespace functions with new runtime functions")
+    '+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+    @it("supports function stubbing with a new function at runtime")
```

Comment:
Will be adding more but could not get the tests for this project running. Have been validating the changes in our project.